### DISCUSSION TITLE
Incorrect klog tags flag in website docs

### DISCUSSION THREAD

**Discussion by vladdeSV** [jotaen/klog#323] [Something else]
Not sure where to report this, since it's only about the website with the docs and not with the software itself.

On the docs page for klog, [this paragraph](https://klog.jotaen.net/#:~:text=Break%20down%20by%20tags) is using the wrong flag

> If you use tags with values and want to see a fine-granular breakdown, use the --value flag.

It should be `--values`.

**Comment by jotaen** [comment]
Oh, thanks for catching this – fixed!
